### PR TITLE
Update unit test expectation

### PR DESCRIPTION
**Pull request by s-hertel**
##### SUMMARY
A collection with an empty/null version can't be installed with `ansible-galaxy`. If a MANIFEST.json contains '' as the version, the collection has been edited after installation or been created with a different tool. I'm not sure that allowing this makes sense, so I updated the test instead of editing the code. Do we want to add a note to the changelog in case someone was doing this?

This only matters for `ansible-galaxy collection list` since other operations with collection artifacts should fail on empty/null versions.

##### ISSUE TYPE
- Test Pull Request

### CODE REVIEW

File: test/units/galaxy/test_collection_install.py [hunk 1]
```diff
@@ -226,17 +226,18 @@ def test_build_requirement_from_path_no_version(collection_artifact, monkeypatch
     mock_display = MagicMock()
     monkeypatch.setattr(Display, 'display', mock_display)
 
-    manifest_path = os.path.join(collection_artifact[0], b'MANIFEST.json')
```
Comment (webknjaz) [inline test/units/galaxy/test_collection_install.py:215]: @s-hertel I think a negative test for `MANIFEST.json` would also be useful.

Comment (s-hertel) [reply test/units/galaxy/test_collection_install.py:215]: Oh, true. Adding that now.

File: test/units/galaxy/test_collection_install.py [hunk 2]
```diff
@@ -235,9 +236,35 @@ def test_build_requirement_from_path_no_version(collection_artifact, monkeypatch
             'dependencies': {}
         }
     })
+
```
Comment (s-hertel) [inline test/units/galaxy/test_collection_install.py]:
```suggestion

```

**Comment by webknjaz** [comment]
absorbed

**Pull request closed without merging**